test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -128,20 +129,26 @@ func getGPUsAvailable(f *framework.Framework) int64 {
 func SetupNVIDIAGPUNode(f *framework.Framework, setupResourceGatherer bool) *framework.ContainerResourceGatherer {
 	logOSImages(f)
 
+	ds := &appsv1.DaemonSet{}
```

Comment:
nit: define a bare variable here:

```
var ds *appsv1.DaemonSet
```

and then assign the DaemonSet at L137 and L145.